py3status/modules/README.md

```diff
@@ -159,6 +159,7 @@ also waiting.
 Configuration parameters:
   - `cache_timeout` How often we refresh this module in seconds *(default 600)*
   - `format` Display format to use
+  - `hide_if_zero` Don't show on bar if True *(default False)*
```

Comment:
Ah rats! I didn't see that. Sorry. Should I fix it?
If not and if that's everything... Well... Thanks for the big help. :-)